pom.xml

```diff
@@ -25,7 +25,7 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.303.3</jenkins.version>
+    <jenkins.version>2.361.4</jenkins.version>
```

Comment:
I missed that there is another change needed in this file in order to complete the update of the parent pom and the base Jenkins version.  That [step](https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/#update-minimum-required-jenkins-version) is described in the [tutorial](https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/).

The [plugin bill of materials](https://www.jenkins.io/doc/developer/tutorial-improve/use-plugin-bill-of-materials/) needs to be updated  from `bom-2.303.x` version `1500.ve4d05cd32975` to `bom-2.361.1` version `1981.v17df70e84a_a_1` as shown at https://github.com/jenkinsci/javadoc-plugin/blob/5f78d449fd15977f14b04cada74b090b67e46acd/pom.xml#L35-L36

Reply:
@MarkEWaite,  You specified In the tutorial that the bom version should be the latest release version but `226.v71211feb_e7e9`  is the latest release --> "correct me if I'm wrong"

Out of my interest, what is the significance of specifying the version in bom?

Reply:
> the bom version should be the latest release version but 226.v71211feb_e7e9 is the latest release

https://github.com/jenkinsci/bom/releases shows that `1981.v17df70e84a_a_1` is the most recent release of the plugin bill of materials. 

> Out of my interest, what is the significance of specifying the version in bom?

The [tutorial](https://www.jenkins.io/doc/developer/plugin-development/dependency-management/#update-the-jenkins-core-bill-of-materials-bom) includes a [video](https://youtu.be/Fev8KfFsPZE?t=807) that gives an overview of the plugin bill of materials.  

New versions of the plugin bill of materials update the versions of plugins that are defined in the bill of materials.  Newer bill of materials versions allow plugins to depend on newer plugin releases without requiring that the plugin track the version of each plugin in its own pom file.

Bill of materials `bom-2.361.x` version `1500.ve4d05cd32975` will list older plugin versions than `bom-2.361.x` version `1981.v17df70e84a_a_1`.  By using the most recent release of the plugin bill of materials, you're choosing to rely on the most recent releases of plugins as the dependencies of this plugin.

Reply:
Sorry for that,  I just got confused!!

Reply:
No problem at all.  Thanks for being involved!